apps/slack/Dockerfile

```diff
@@ -4,12 +4,15 @@ RUN apk add --update --no-cache openssh build-base python3 bash git && rm -rf /v
 
 WORKDIR /app
 
-COPY package.json yarn.lock frontend/package.json lambda/package.json /app/
+#package.json was not brought over from the other repo, in fact it's part of a yarn workspaces set up that we might want to move away from
+COPY package.json package-lock.json frontend/package.json lambda/package.json /app/
```

Comment:
Ah right I see. So package.json doesn't exist. Did you mean to introduce `package-lock.json` here as well? (which also doesn't exist)